pkg/plugins/policies/meshloadbalancingstrategy/plugin/v1alpha1/plugin.go

```diff
@@ -294,9 +324,20 @@ func (p plugin) configureListener(
 	})
 }
 
-func (p plugin) configureCluster(c *envoy_cluster.Cluster, lbConf *api.LoadBalancer) error {
-	if lbConf == nil {
-		return nil
+func (p plugin) configureCluster(c *envoy_cluster.Cluster, config api.Conf) error {
+	if shouldUseLocalityWeightedLb(config) {
+		if err := (&xds.LocalityWeightedLbConfigurer{}).Configure(c); err != nil {
+			return err
+		}
+	}
+	if config.LoadBalancer != nil {
```

Comment:
why this if? 🤔

Reply:
Previously we were checking this not to configure LB, now I've changed to the parent object so wanted to check if we need to run a plugin.